plugins/tiddlywiki/markdown/wrapper.js

```diff
@@ -74,7 +74,8 @@ function transformNode(node) {
 		}
 		return widget;
 	} else {
-		return {type: "raw", html: node};
+		return $tw.wiki.parseText("text/vnd.tiddlywiki", node, {parseAsInline: true}).tree[0].\
+				extend({type: "text", text: " "});
```

Comment:
Hey @pmario sorry I thought that was working - but wasn't. Agreed no hardcoding should be done. Could you please help me - I'm excited about having macros in Markdown. But I'm getting my spaces trimmed between elements! Do you think this (fixed) implementation would be acceptable? I know it's been requested at #2887 and it would be oh so useful.

I'm glad to be of service and thank you for your patience as I get up to speed with this nice codebase!
Thanks!